index.js

```diff
@@ -212,7 +217,14 @@ NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
     return null
   }
 
-  return this.transform(code, {filename: filename, relFile: relFile})
+  var ext = path.extname(filename)
```

Comment:
Note that `extname` only returns the *last* extension. Looping through the registered transforms to see if they match the filename would let us support `.foo.bar`.